tools/proto_sync.py

```diff
@@ -3,18 +3,68 @@
 # Diff or copy protoxform artifacts from Bazel cache back to the source tree.
 
 import glob
+import os
 import re
 import shutil
+import string
 import subprocess
 import sys
 
+# This maps from .proto import directive path to the Bazel dependency path for
```

Comment:
Can this mapping go to another file together with https://github.com/envoyproxy/envoy/blob/master/api/bazel/api_build_system.bzl#L13? Probably in a separate bzl file that you can parse from Python as well?

Reply:
Good idea! Done.